i18n/shuffle/fwk/service/teams/EditTeamService.properties

```diff
@@ -18,4 +18,24 @@ button.add=Add
 text.title=Team Editor
 text.level=Level: 
 text.name=Name: 
-text.type=Type: 
\ No newline at end of file
+text.type=Type: 
+tooltip.type=Filter by type
+tooltip.level=Only show species with this level or higher
+tooltip.name=Filter by name
+tooltip.megafilter=Only show species with Mega evolutions
+tooltip.effectfilter=Filter by effect
+tooltip.makedefault=Make the current team the default team for this stage
```

Comment:
Not for this stage, but for this type of stage - it saves the current team as the team for the generic stage for this current stage's type. I.e. if you are on Snorlax and save as default, then the current team is saved to the "NORMAL" stage's team.